### ISSUE TITLE
Command palette closes if you click on the palette's scrollbar

### ISSUE THREAD

**Issue by Eskander88** [atom/atom#17236]
*Edit by @rsese to add the issue template info*

### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Clicking the scrollbar of the command palette (to scroll up and down the list) closes the command palette.

### Steps to Reproduce

1. `atom --safe`
2. Open the command palette 
3. Click the scrollbar to try and scroll up or down

**Expected behavior:** [What you expect to happen]

Be able to click/hold the scrollbar to drag up and down and to click on the scrollbar to move up and down.

**Actual behavior:** [What actually happens]

Clicking anywhere on the scrollbar closes the command palette.

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions

```
atom -v
Atom    : 1.28.0-dev-59cb1dd50
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
```

macOS 10.12.6

### Additional Information

x-ref: https://github.com/atom/command-palette/issues/4

---

I press `ctrl-shift-p` and ATOM is showed list of `command-pallete`. I can input keyword for seacrh commands. When i click on scroll bar (vertical line) for scroll down or up then `command-pallete` is hidden.

I can move list with button `up` or `down` only

Tried with Atom x64-1.25.1, x64-1.27.0-beta1 – Windows 10 x64

**Comment by rsese** [comment]
Thanks for moving this from the message board @Eskander88 :bow:  I went ahead and updated the issue body to use the issue template - it would be super helpful for us if you can use the issue template for future issues.

**Comment by Eskander88** [comment]
thank for showing example of using template

**Comment by lee-dohm** [comment]
@simurai Is this something you could take a look at? Is it possible that this is CSS related?

**Comment by maxbrunsfeld** [comment]
This will be fixed in Atom 1.31.
